Protobuf-C++.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name     = 'Protobuf-C++'
-  s.version  = '3.20.1-rc1'
+  s.version  = '3.21-dev'
```

Comment:
That's a good point -- I was thinking it might be a good way to demonstrate what changes the update_version script produces, but I can just link to the second PR to keep this easier to review.